Dockerfile

```diff
@@ -88,9 +123,13 @@ ENV POETRY_VERSION="2.1.3"
 
 WORKDIR ${APP_PATH}
 
-RUN set -eux; \
-    python -m venv ${PHYLUM_VENV}; \
-    ${PHYLUM_VENV_PIP} install --no-cache-dir --upgrade pip setuptools
+RUN python -m venv ${PHYLUM_VENV}
+# Isolate the use of the `PHYLUM_TOKEN` secret to a single RUN command so as not
+# to leak the value in logs with `set -eux` command
```

Comment:
Does this work? Instead of leaking the value in build logs, the value is being saved into an image layer, and then that layer is being saved into GitHub via the cache, even if we're ultimately publishing a different image that does not contain this layer.